devops/eslint-config/index.js

```diff
@@ -0,0 +1,40 @@
+// This config is used globally at the root of the repo, so it should be as thin
+// as possible with rules that we absolutely require across all projects.
+module.exports = {
+  env: {
+    es6: true
+  },
+  globals: {
+    Atomics: 'readonly',
+    SharedArrayBuffer: 'readonly'
+  },
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true
+    },
+    ecmaVersion: 2019,
+    sourceType: 'module'
+  },
+  extends: [
+    'plugin:react/recommended',
+    'standard'
```

Comment:
`standard` is a styleguide package, we probably should use `eslint:recommended` instead

Reply:
Or alongside if you intended to use `standard`

Reply:
If I include `standard` as first rule-set, followed by `eslint:recommended`, we get benefit of both with the eslint:recommended rules taking precedence over standard?

Reply:
I'm actually going to remove any styleguide packages, because we need prettier for consistent code formatting.

Reply:
> If I include standard as first rule-set, followed by eslint:recommended, we get benefit of both with the eslint:recommended rules taking precedence over standard?

Yes, that's my understanding.

> I'm actually going to remove any styleguide packages, because we need prettier for consistent code formatting.

This makes sense, since prettier eslint plugin will disable any style rules anyway.

Reply:
Wouldn't `prettier eslint plugin` only disable the rules conflicting with prettier? So we would not get _all_ the benefits of the style guide, but still keep relevant checks like `eqeqeq`

Reply:
Check like `eqeqeq` would be inside `eslint:recommended`, not `standard`. We could keep `standard` however, since looking at the rules it enables (https://github.com/standard/eslint-config-standard/blob/master/eslintrc.json), some may not be covered by `prettier` since they aren't all code formatting related.

Reply:
yes also I think standard works better for vanilla javascript. Its the standardjs guide if I'm not mistaken